tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/ejb/ejb/MessageDrivenEJBExecutionTime.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.hal.testsuite.test.runtime.ejb.ejb;
+
+import javax.ejb.ActivationConfigProperty;
+import javax.ejb.MessageDriven;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import javax.jms.TextMessage;
+
+import org.jboss.logging.Logger;
+
+@MessageDriven(activationConfig = {
+    @ActivationConfigProperty(propertyName = "destinationLookup", propertyValue = "queue/MessageDrivenExecutionTimeQueue"),
+    @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Queue"),
+    @ActivationConfigProperty(propertyName = "acknowledgeMode", propertyValue = "Auto-acknowledge")})
+public class MessageDrivenEJBExecutionTime implements MessageListener {
+
+    private final Logger logger = Logger.getLogger("MessageDrivenEJBExecutionTime");
+
+    @Override
+    public void onMessage(Message message) {
+        TextMessage msg;
+        try {
+            Thread.sleep(5000);
```

Comment:
It's quite long. How many time is this sleep called? Such long sleep appears 9 times in the code of this pull request.